### PR TITLE
add folder commands for custom config

### PR DESCRIPTION
**Pull request by seniorivn**

### CODE REVIEW

**Comment by seniorivn** [comment]
@john-preston it seems you are the most active participant. Could you merge this small fix?

**Comment by john-preston** [comment]
@seniorivn 

image

Why didn't you add the rest at the bottom part?

**Comment by seniorivn** [comment]
@john-preston 
good question, will fix.
Which reminds me: Shouldn't it be automated? Exposing all Commands to config somehow?

File: Telegram/SourceFiles/core/shortcuts.cpp [hunk 1]
```diff
@@ -104,6 +118,20 @@ const auto CommandNames = base::flat_map<Command, QString>{
 	{ Command::ChatFirst     , qsl("first_chat") },
 	{ Command::ChatLast      , qsl("last_chat") },
 	{ Command::ChatSelf      , qsl("self_chat") },
+	
+	
+	{ Command::FolderPrevious ,  qsl("previous_folder")   },
+	{ Command::FolderNext     ,  qsl("next_folder")       },
+	{ Command::ShowAllChats   , qsl("all_chats")   },
+	                                                   
+	{   Command::ShowFolder1       , qsl("folder1")    },
```
Comment (john-preston) [inline Telegram/SourceFiles/core/shortcuts.cpp:127]: It looks like you've used different indentation here, three spaces instead of one elsewhere.

Comment (Aokromes) [reply Telegram/SourceFiles/core/shortcuts.cpp:127]: https://github.com/seniorivn/tdesktop/blob/patch-1/Telegram/SourceFiles/core/shortcuts.cpp#L57 the indentation on the whole file looks to be very random xd

**Comment by john-preston** [comment]
Whatever. Thanks.

**Pull request merged**